code-execution-engines/langchain-code-execution-engine-benchmark/src/test/java/dev/langchain/code/benchmark/JMHBenchmarkJavaScriptExecution.java

```diff
@@ -0,0 +1,70 @@
+package dev.langchain.code.benchmark;
+
+
+import dev.langchain4j.code.CodeExecutionEngine;
+import dev.langchain4j.code.graalvm.GraalVmJavaScriptExecutionEngine;
+import dev.langchain4j.code.v8.V8JavaScriptExecutionEngine;
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.CommandLineOptionException;
+import org.openjdk.jmh.runner.options.CommandLineOptions;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+import java.util.concurrent.TimeUnit;
+
+@State(Scope.Benchmark)
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+@Fork(1)
+@Warmup(iterations = 1)
```

Comment:
is 1 iteration enough?

Reply:
basically how V8 and graal work internally I assume yes

but here is results for 5 iteration (you can try) and I will commit 5 iterations warmup


Benchmark                                       Mode  Cnt  Score   Error  Units
JMHBenchmarkJavaScriptExecution.benchmarkGraal  avgt    2  1.234          ms/op
JMHBenchmarkJavaScriptExecution.benchmarkV8     avgt    2  0.674          ms/op



NOTE this is also with graal jdk with graal jit compiler, I assume in regular openJDK the V8 is much much faster